integreat_cms/core/management/commands/reset_deepl_budget.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from datetime import datetime
+
+from ....cms.models import Region
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+
+class Command(LogCommand):
+    """
+    Management command to reset DeepL budget
+    """
+
+    help = "Reset DeepL budget"
+
+    def handle(self, *args, **options):
+        """
+        Try to run the command
+        """
+
+        current_month = datetime.now().month - 1
+
+        if not (regions := Region.objects.filter(deepl_renewal_month=current_month)):
+            self.print_info(
+                "✔ There is no region whose DeepL budget needs to be reset."
+            )
+        else:
+            for region in regions:
+                region.deepl_budget_used = 0
+                region.deepl_midyear_start_month = current_month
```

Comment:
I think this should be reset to `None`, which is tthe default for non-midyear-starting regions, which the regions turns into after passing `deepl_renewal_month` for the first time.

Reply:
It is so specified in the issue description. How do you think @timoludwig ?

Reply:
Ah sorry, this was a mistake in the issue description. It won't make any difference in the calculation, but I agree with @charludo that it makes more sense to have the midyear start disabled (by setting it to `None`) rather than enabling it and set it to the same month as the regular renewal month...

Reply:
Applied :slightly_smiling_face: